pkg/controller/releasemanager/state.go

```diff
@@ -224,6 +224,9 @@ func PendingTest(ctx context.Context, deployment Deployment) (State, error) {
 	if externalTestStatus == ExternalTestStarted {
 		return testing, nil
 	}
+	if err := deployment.sync(ctx); err != nil {
```

Comment:
I wasn't sure if this was needed or not, to get this to stick: https://github.com/Medium/picchu/blob/set-external-test-last-updated/pkg/controller/releasemanager/incarnation.go#L383